iothub/device/src/DirectMethod/DirectMethodResponse.cs

```diff
@@ -11,11 +11,13 @@ namespace Microsoft.Azure.Devices.Client
     /// </summary>
     public class DirectMethodResponse
     {
+
         /// <summary>
         /// Initializes an instance of this class.
         /// </summary>
-        public DirectMethodResponse()
+        public DirectMethodResponse(int status)
```

Comment:
Making status a mandatory field for direct method response

Reply:
It was mandatory for PnP, but not otherwise mandatory, right? Or was it?

Reply:
The public document doesn't call out the status being required. However, a direct method response without an associated status looks logically incomplete. What do you think?
For an additional data point, our public API had also expected the direct method response to accompany a status code: https://github.com/Azure/azure-iot-sdk-csharp/blob/main/iothub/device/src/DirectMethod/MethodResponse.cs

Reply:
I agree that as a best practice this is a good idea.